src/core/species_mod.F90

```diff
@@ -46,7 +46,7 @@ module species_mod
       integer :: drydep_index         !< drydep index in drydep array
```

Comment:
Do we know if Doxygen cares about the `!<`?

Reply:
I don't